CONTRIBUTING.md

```diff
@@ -1,16 +1,56 @@
-The specification and schema definition, including application examples in the aas-spec repository must be particularly compliant with this.
-However, we invite the community to review, report and fix the specification and schema definitions, including application examples. 
-Therefore, we demand a defined procedure for the contribution in this repository.
+Thank you for your interest in the specifications of the Asset Administration Shell!
 
-[Asset Administration Shell - Part 1]: https://www.plattform-i40.de/PI40/Redaktion/EN/Standardartikel/specification-administrationshell.html
+## Industrial Digital Twin Association (IDTA)
+
+Governance of the specifications of the Asset Administration Shell is done in the working group *Open Technology* 
+of the [IDTA](https://industrialdigitaltwin.org/en/)
+
+However, we invite the community to review, report and fix the specification 
+and schema definitions, openAPI definition including application examples. 
+
+## Naming Convention for github repos
+
+For all github repositories that contain a part belong to the document series "Specification Asset Administration Shell" 
+the name of the github repository is starting with **aas-specs**.
+
+
+## Issue Handling
+
+It is requested that every pull request is assigned to at least one related issue.
+
+There are two types of issues:
+- Minor changes (simple failures, typos, *etc.*) and additional content (more examples, etc.) can be accepted straight away after a brief review by the responsible reviewers.
+  The corresponding issue will be labeled with **accepted** or **accepted in principle**
+
+- Major changes must first be reviewed and approved by the Workstream "Specification Asset Administration Shell" within the "OpenTechnology" Working "Asset Administration Shell" of the  [IDTA].
+  Corresponding issues will be labeled with **requires workstream approval** and - after review in the Workstream - with **accepted** or **accepted in principle**.
+  If they are not accepted they will be closed without resolution.
+
+Additionally the following classification by labeling is recommended:
+- **specification** (major): the issue has impact on the implementation of the specifciation, e.g. adding new attribute to class, adding new API-operations, depracting element etc. 
+- **documentation**: the issue has only impact on the editorial part of the specification but not impact on the implementation
+- **process**: the issue has no impact on the specification but requires changes or improvements of the process as e.g. defined in this contribution guide
+
+specific for IDTA-01001 Metamodel:
+- **aas-core** (major): the issues has impact on the generation of the schema etc. but not impact on the specification itself.
+    Note: Every issue that has impact on the specification (label *specification* also has impact on aas-core).
```

Comment:
The second half of the sentence is hard to understand. But even if these issues impact the specification, why should we mention aas-core here?